packages/resolvers-composition/src/resolvers-composition.ts

```diff
@@ -26,16 +26,16 @@ function resolveRelevantMappings<Resolvers extends Record<string, any> = Record<
   path: string,
   allMappings: ResolversComposerMapping<Resolvers>
 ): string[] {
-  const splitted = path.split('.');
```

Comment:
https://www.merriam-webster.com/dictionary/splitted#:~:text=archaic%20past%20tense%20of%20split

obviously, I can drop if people are attached to this spelling